nilearn/datasets/struct.py

```diff
@@ -106,38 +102,130 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
-    mni152_template : nibabel object corresponding to the template
+    mni152_template : Nifti1Image, image representing the re-sampled
+        whole-brain template
 
     References
     ----------
     .. footbibliography::
 
     """
-    return check_niimg(MNI152_FILE_PATH)
+
+    brain_template = check_niimg(MNI152_FILE_PATH)
+    new_brain_template = new_img_like(
+        brain_template, get_data(brain_template).astype('uint16'))
+
+    # Change the resolution of the template
+    if resolution != 1:
+        new_brain_template = resampling.resample_img(new_brain_template,
+                                                     np.eye(3) * resolution)
+
+    return new_brain_template
+
+
+def load_mni152_gm_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    gray-matter MNI152 template, originally distributed with FSL, and
+    re-samples it using a different resolution, if specified.
+
+    .. versionadded:: 0.8.0
+
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+    Returns
+    -------
+    gm_mni152_template : Nifti1Image, image representing the resampled
+        gray-matter template
+
+    """
+
+    gm_template = check_niimg(GM_MNI152_FILE_PATH)
+    new_gm_template = new_img_like(
+        gm_template, get_data(gm_template).astype('uint16'))
+
+    # Change the resolution of the template
+    if resolution != 1:
+        new_gm_template = resampling.resample_img(new_gm_template,
+                                                  np.eye(3) * resolution)
+
+    return new_gm_template
+
+
+def load_mni152_wm_template(resolution=2):
+    """This function takes the skullstripped 1mm-resolution version of the
+    white-matter MNI152 template, originally distributed with FSL, and
+    re-samples it using a different resolution, if specified.
+
+    .. versionadded:: 0.8.0
+
+    Parameters
+    ----------
+    resolution: int, optional, Default = 2
+        If resolution is different from 1, the template is re-sampled with the
+        specified resolution.
+
+    Returns
+    -------
+    wm_mni152_template : Nifti1Image, image representing the resampled
+        white-matter template
+
+    """
+
+    wm_template = check_niimg(WM_MNI152_FILE_PATH)
+    new_wm_template = new_img_like(
+        wm_template, get_data(wm_template).astype('uint16'))
+
+    # Change the resolution of the template
+    if resolution != 1:
+        new_wm_template = resampling.resample_img(new_wm_template,
+                                                  np.eye(3) * resolution)
+
+    return new_wm_template
 
 
-def load_mni152_brain_mask():
-    """Load brain mask from MNI152 T1 template
+def load_mni152_brain_mask(resolution=2, threshold=0.2):
+    """Load mask from the whole-brain MNI152 T1 template.
 
-    .. versionadded:: 0.2.5
+    .. versionadded:: 0.8.0
```

Comment:
This line should be changed to `.. versionadded:: 0.8.1` and be placed under each new argument (resolution and threshold here).